src/modules/app/containers/Switcher.js

```diff
@@ -64,67 +64,53 @@ type PropsType = {
  */
 export class Switcher extends React.Component<PropsType> {
   /**
-   * Renders a failure component if the payload contains an error or a LoadingSpinner if the data is currently being fetched
-   * @param payload The payload to check for errors or fetching process
+   * Renders a failure component if a payload contains an error or a LoadingSpinner if the data is still being fetched
+   * @param payloads The payloads to check for errors or fetching process
    * @return {*}
    */
-  static renderFailureLoadingComponents = (payload: Payload<any>): Node => {
-    if (payload.error) {
-      return <FailureSwitcher error={payload.error} />
-    } else if (payload.isFetching || !payload.data) {
+  static renderFailureLoadingComponents = (payloads: Array<Payload<any>>): ?Node => {
```

Comment:
You should rather use `mixed` instead of `any`. Best would, of course, be to use neither of them.

Reply:
This is a perfect example to use `Payload<*>` here.
http://sitr.us/2015/05/31/advanced-features-in-flow.html#existential-types